roles/api-server/files/api-server/templates/deployment.yaml

```
@@ -27,6 +27,12 @@ spec:
             value: "{{  .Values.frontend.url }}"
           - name: USER_ADMITTANCE
             value: "{{ .Values.api.userAdmittance }}"
+{{- if eq .Values.mail.mailProvider "mailjet" }}
+          - name: MJ_APIKEY_PUBLIC
+            value: "{{ .Values.mail.mjApiKeyPublic }}"
+          - name: MJ_APIKEY_PRIVATE
+            value: "{{ .Values.mail.mjApiKeyPublic }}"
+{{- end -}}
```

The trailing trimming `-` produced this error: https://github.com/metal-stack-cloud/ansible-roles/pull/16#issuecomment-1517837701

```suggestion
{{- end }}
```

---

@vknabel please push your fixes to this PR that we can get this out and released